web_frontend/osmaxx/countries/utils.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from django.contrib.gis.geos import MultiPolygon, Polygon, GEOSGeometry
+
+from osmaxx.countries._settings import POLYFILE_LOCATION
+
+POLYFILE_ENDING = '.poly'
+
+
+def get_polyfile_name_to_file_mapping():
+    polyfile_mapping = {}
+    for possible_polyfile in os.listdir(POLYFILE_LOCATION):
+        if possible_polyfile.endswith(POLYFILE_ENDING):
+            name = possible_polyfile.split(POLYFILE_ENDING)[0]
+            polyfile_mapping[name] = possible_polyfile
+    return polyfile_mapping
+
+
+def polyfile_to_geos_geometry(relative_polygon_file, simplify_tolerance=None):
+    with open(os.path.join(POLYFILE_LOCATION, relative_polygon_file)) as poly_file:
+        poly = parse_poly(poly_file.readlines())
+        if simplify_tolerance:
+            poly = poly.simplify(tolerance=simplify_tolerance, preserve_topology=True)
+            # simplifying can lead to a polygon. Assure, it stays a multipolygon.
+            if poly and isinstance(poly, Polygon):
+                poly = MultiPolygon(poly)
+    return GEOSGeometry(poly)
+
+
+def parse_poly_string(poly_string):
+    return parse_poly(poly_string.split(os.linesep))
```

Comment:
Should this be `os.linesep` or a fixed line separator as found in the Polygon Filter files we download?

Or should we split at anything that can be interpreter as a line separator (Python's "universal newlines" notion) with [`poly_string.splitlines()`](https://docs.python.org/3/library/stdtypes.html#str.splitlines)?